### DISCUSSION TITLE
Force push update to watch?

### DISCUSSION THREAD

**Discussion by cwpeterson** [Navid200/xDrip#121] [Watches]
I updated to the most recent nightly and got the message that it was sending the update to my watch. Unfortunately I had adb debugging disabled. So now how can I force it to push the update when my watch is ready to receive it?

**Comment by Navid200** [comment]
Does this help?
https://github.com/Navid200/xDrip/wiki/WearOS-Instructions

**Reply by cwpeterson** [reply]
Yes thanks. I will try again when I'm home and have access to wifi.
